src/app/shared/services/account/account.service.spec.ts

```diff
@@ -99,4 +125,36 @@ describe('AccountService', () => {
       expect(error).toEqual(expectedError);
     }
   });
+
+  it('should handle successful email verification', async () => {
+    const { instance } = shallow.createService();
+
+    const account = new AccountVO({
+      primaryEmail: 'test@permanent.org',
+      fullName: 'Test User',
+      keepLoggedIn: true,
+      emailStatus: 'status.auth.unverified',
+    });
+
+    instance.setAccount(account);
+
+    await instance.verifyEmail('sampleToken');
+    expect(instance.getAccount().emailStatus).toBe('status.auth.verified');
```

Comment:
```suggestion
    expect(instance.getAccount().emailStatus).toBe('status.auth.verified');
    expect(instance.getAccount().keepLoggedIn).toBeTrue();
```
Let's add a check for the `keepLoggedIn` behavior since that's what this particular ticket is all about. These tests otherwise just check for email/phone verification, and so pass even when copied over to the main branch. I'd like to see these tests fail if we run them if the specific bug addressed in this ticket isn't fixed, if that makes sense.